wasp/widgets.py

```diff
@@ -326,12 +343,12 @@ def draw(self, message):
         wasp.watch.drawable.blit(
             icons.yes_button,
             _yes_button_x_coord,
-            _yes_button_y_coord,
+            _yes_button_y_coord
```

Comment:
Again, no need to modify unrelated code.

Reply:
This was left as an artifact of the confirmation dialog theming, will remove